src/Microsoft.AspNet.Identity.EntityFramework/IdentityDbContext.cs

```diff
@@ -33,9 +33,14 @@ protected override void OnModelCreating(ModelBuilder builder)
             builder.Entity<TUser>(b =>
                 {
                     b.Key(u => u.Id);
+                    b.Index("NormalizedUserName", "NormalizedEmail");
```

Comment:
This is me misusing the API, I thought I was specifying 2 indexes, oops, I guess what I meant to do is to have two calls to Index...will update